engine/apps/webhooks/models/webhook.py

```diff
@@ -152,7 +152,9 @@ class Webhook(models.Model):
     http_method = models.CharField(max_length=32, default="POST", null=True)
     trigger_type = models.IntegerField(choices=TRIGGER_TYPES, default=TRIGGER_ESCALATION_STEP, null=True)
     is_webhook_enabled = models.BooleanField(null=True, default=True)
+    # NOTE: integration_filter is deprecated (to be removed), use filtered_integrations instead
     integration_filter = models.JSONField(default=None, null=True, blank=True)
+    filtered_integrations = models.ManyToManyField("alerts.AlertReceiveChannel", related_name="webhooks")
```

Comment:
naming suggestion: maybe just `integrations`?

Reply:
I thought about that, but it seemed to me that having `filtered_integrations = []` is clearer that we don't filter by integration (and then this applies to all) when the webhook is triggered than `integrations = []`? Still not super happy with the name, so I'm open to suggestions :-)

Reply:
makes sense 👍